etna/pipeline/pipeline.py

```diff
@@ -54,9 +56,16 @@ def _forecast(self) -> TSDataset:
         if self.ts is None:
             raise ValueError("Something went wrong, ts is None!")
 
-        if isinstance(self.model, DeepBaseModel):
-            future = self.ts.make_future(future_steps=self.model.decoder_length, tail_steps=self.model.encoder_length)
-            predictions = self.model.forecast(ts=future, horizon=self.horizon)
+        if isinstance(self.model, NonPredictionIntervalContextRequiredAbstractModel) or isinstance(
```

Comment:
I guess we need ContextRequiredModel type

Reply:
I'll try to find how to do it.